airflow/kubernetes/pod_generator.py

```diff
@@ -412,16 +415,13 @@ def deserialize_model_file(path: str) -> k8s.V1Pod:
         """
         :param path: Path to the file
         :return: a kubernetes.client.models.V1Pod
-
-        Unfortunately we need access to the private method
```

Comment:
This doctoring was missed to be moved during earlier refactoring https://github.com/apache/airflow/pull/10756